### PR TITLE
docs: fix typos

### PR DESCRIPTION
**Pull request by Lioness100**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "X" next to an item)

[X] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-change-proposal.md))
[ ] Add autofix to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:

#### What changes did you make? (Give an overview)

Fixed a few random typos (this was a trial run for [a CLI I made](https://github.com/Lioness100/rspell))

### CODE REVIEW

File: lib/rules/logical-assignment-operators.js
Comment (amareshsm) [inline lib/rules/logical-assignment-operators.js:115]:
```suggestion
 * falseness checks:   !value, !Boolean(value)
```

Comment (Lioness100) [reply lib/rules/logical-assignment-operators.js:115]: That seems a bit misleading- the point of the name falsy is that it's not exactly false. Also, if we go with "falseness", we would have to change "truthiness" to "truthness", right?

File: docs/src/rules/arrow-parens.md
Comment (Rec0iL99) [inline docs/src/rules/arrow-parens.md:136]: `falsy` can sometimes be written as `falsey` too according to [MDN](https://developer.mozilla.org/en-US/docs/Glossary/Falsy)

Comment (Lioness100) [reply docs/src/rules/arrow-parens.md:136]: I changed it to falsy just for consistency with the rest of the codebase, which uses falsy over falsey.

Comment (Rec0iL99) [reply docs/src/rules/arrow-parens.md:136]: If that's the case you can resolve my change. Thanks

File: tools/fuzzer-runner.js
Comment (mdjermanovic) [inline tools/fuzzer-runner.js:50]:
```suggestion
        "Fuzzing rules [:bar] :percent, :elapseds elapsed, eta :etas, errors so far: :elapsedErrors",
```

This was intentional, to print `s` (as short for seconds) after [`:elapsed`](https://github.com/visionmedia/node-progress#tokens). For example, `19.7s elapsed`

**Review by mdjermanovic — CHANGES_REQUESTED** [review]

File: lib/rules/logical-assignment-operators.js
Comment (amareshsm) [reply lib/rules/logical-assignment-operators.js:115]: I haven't seen `falsiness` word being used so I suggested `falseness` if it is used then I am ok with the change. Can you pls check and confirm this?

Comment (Lioness100) [reply lib/rules/logical-assignment-operators.js:115]: https://en.m.wiktionary.org/wiki/falsiness

File: tools/fuzzer-runner.js
Comment (Lioness100) [reply tools/fuzzer-runner.js]: Oops, sorry about that.

**Review by amareshsm — APPROVED** [review]
LGTM, Thanks!
Just waiting for one more approval.

**Comment by Lioness100** [comment]
My pleasure!

**Review by snitin315 — APPROVED** [review]
LGTM, thanks!

**Review by mdjermanovic — APPROVED** [review]
LGTM, thanks!

**Pull request merged**
